src/browser.js

```diff
@@ -198,7 +198,7 @@ function edgeSupported () {
     return prom;
 }
 
-var regItemPattern = /\s*\(Default\)\s+(REG_SZ)\s+([^\s].*)\s*/;
+var regItemPattern = /\s*\(.*\)\s+(REG_SZ)\s+([^\s].*)\s*/;
```

Comment:
I think this should use `[^)]+` or `.+?` instead of `.*`. Otherwise the match could extend over the closing parenthesis.